src/mathf/dimensions-2d.ts

```diff
@@ -0,0 +1,60 @@
+import { Vector } from './vector';
+import { SCROLL_ELEMENT } from '../dom/scroll-element';
+import { ROOT_ELEMENT } from '../dom/root-element';
+
+export class Dimensions2d extends Vector {
```

Comment:
This is tough one.

So in my mind, mathf tends to be pure math functions.  But this class is technically, DOM related so I find it more suitable under /dom and perhaps something like DomVector something.  WDYT?

Reply:
Created DOM variants.